source/ht_camera.cpp

```diff
@@ -43,6 +43,16 @@ namespace Hatchit {
             IRenderer::Instance->RegisterCamera(*this, m_layerflags);
         }
 
+        Math::Matrix4 Camera::GetView()
+        {
+            return m_view;
```

Comment:
Can we return by const ref instead of by value?  Don't want to be unnecessarily copying data